_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,428 @@
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 \
+      --no-install-recommends \
+    && apt-get remove -y google-chrome-stable \
+    && rm -rf /var/lib/apt/lists/*
+
+ENTRYPOINT ["/usr/local/bin/npx", "aws-lambda-ric"]
+
+ENV HOME="/tmp"
+CMD [ "/function/src/index.handler" ]
+
+```
+
+Next, run the following build command within the same directory that contains the Dockerfile:
+
+```
+docker build -t opensearch-reporting-cli .
+```
+
+### Step 2: Create a private repository with Amazon ECR
+
+Follow the instructions in the Amazon ECR user guide to create a repository with the name `opensearch-reporting-cli`. For instructions, see [Getting started with Amazon ECR using the AWS Management Console](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-console.html).
+
+In addition to the Amazon EC2 instructions, you need to make the following adjustments for the Reporting CLI to function properly:
+
+Navigate to the test function **Configuration** tab to modify two required settings.
+- Make sure to set **Timeout** to at least 5 minutes to allow the reporting CLI to generate the report.
+- Change the setting for **Ephemeral storage** to 1024MB. The default setting is not a sufficient storage amount.
+{: .note }
+
+### Step 3: Run the push commands
+
+You need to get several commands from the AWS EC2 Console to run within the Dockerfile directory, as shown in the following image:
+
+![Amazon ECR console view push commands]({{site.url}}{{site.baseurl}}/images/dashboards/push-commands.png)
+
+1. In the AWS ECR console, go to **Repositories** and choose **opensearch-reporting-cli**.
+1. Choose **view push commands**.
+1. In **Push commands for opensearch-reporting-cli**, copy and run each command sequentially in the Dockerfile directory.
+
+If you run the push commands from a directory that does not contain the Dockerfile, you will get an error.
+{: .note }
+
+### Step 4: Create a lambda function with the container image
+
+1. In **Select container image**, select the container `opensearch-reporting-cli` from the list, and then choose **Select image**.
+1. Specify a name for the function.
+1. In **Architecture**, choose **x86_64**.
+1. Choose **Create function**.
+1. Go to **Lambda function > Configuration > General configuration> Edit timeout** and set the timeout in lambda to 5 minutes.
+1. Set the memory size to at least 1024MB.
+1. Next, test the function either by providing fixed values or variable values in a JSON file.
+
+- If the function contains fixed values, such as email address you do not need a JSON file. You can specify an environment variable in AWS Lambda.
+- If the function takes a variable key-pair value, then you need to specify the values in a JSON file.
```

Comment:
1. Does JSON file mean the JSON object in POST request body? If so i think "file" might be confusing because it doesn't need to be saved as a file on disk
2. Does it follow the same naming convention as environment variables?

Reply:
> 1. Does JSON file mean the JSON object in POST request body? If so i think "file" might be confusing because it doesn't need to be saved as a file on disk

Yes, file should be removed. 

> 2. Does it follow the same naming convention as environment variables?

follows same naming convention as command options